test/plugins/storage.js

```diff
@@ -67,4 +74,23 @@ describe("Image storage", function() {
 			done();
 		});
 	});
+
+	it("should lookup correct extension type", function(done) {
+		const message = this.irc.createMessage({
+			text: "http://localhost:9003/svg-preview",
+		});
+
+		this.app.get("/svg-preview", function(req, res) {
+			res.send("<title>test title</title><meta property='og:image' content='http://localhost:9003/logo.svg'>");
```

Comment:
It's a detail but in tests that include HTML, I tend to invert the quotes, to always have double-quotes for HTML attributes. It doesn't matter here, but when testing rendering/parsing, this consistency is rather helpful.